contrib/for-tests/mount-tester/mt.go

```diff
@@ -152,3 +160,41 @@ func readWriteNewFile(path string) error {
 
 	return readFileContent(path)
 }
+
+func readFileContentInLoop(path string) error {
+	if path == "" {
+		return nil
+	}
+	var content []byte
+	//Expected content "mount-tester new file\n", length (22), retry during 240 second
+	content = TestFileContent(path, 22, 240)
+
+	fmt.Printf("content of file %q: %v\n", path, string(content))
+
+	return nil
+}
+
+func TestFileContent(filePath string, bytesExpected int, retryDuration int) []byte {
```

Comment:
@dalanlan This function doesn't need to be exported, rename to `testFileContent`